jolly-jellyfish/src/page_maker/templates/registration/register.html

```diff
@@ -0,0 +1,18 @@
+{% extends "page_maker/base.html" %}
+{% block content %}
+   <div>
+       <h2>Sing Up</h2>
```

Comment:
```suggestion
       <h2>Sign Up</h2>
```
It did sound fun though...